### ISSUE TITLE
DNS connections

### ISSUE THREAD

**Issue by GambogeGoose** [m2049r/xmrwallet#507]
In last phase of a monero payment I noticed a high number of connections from my smartphone towards several hosts.
Those were DNS requests towards many servers that my firewall blocked, including U.S.A. DoD addresses.

As soon as I allowed that traffic, I was able to complete the transaction.
Why is this unwanted traffic present?

Where are all the contacted IPs taken from?
There is a way to choose which "backup" IP have the app to use?

**Comment by m2049r** [comment]
i seriously doubt this to be true. there should only be ONE connection - to the node you have selected.
please give me info which node you use and which hosts/ports your smartphone is trying to connect to.
is it different when you use the official CLI or GUI on a PC?

**Comment by GambogeGoose** [comment]
I am sorry for a reply this late.
I actually ignored this issue and continued to use monerujo.

Even if the version has changed and some time has passed, I will try to do the same things and use the same approaches as before, making sure to record what I need to continue this issue.

What we know by now is:
- I was not able to complete a transaction blocking several DNS requests;
- I was able to complete the transaction allowing that traffic;
- I used a node that was in may LAN, where also the smartphone was connected.

It could also have been just a coincidence that strange DNS traffic took place while I tried to make a payment through Monerujo.

I will try to test all of these hypotheses:

- **Firewall**
Maybe the firewall rules were too strict or poorly configured.

- **Local Node**
I actually don't know yet what to test on the local node, but I will try with an external node either.
The local node has outbound traffic allowed and didn't precluded me any transaction.
The blockchain is updating correctly.

- **Operating System**
Maybe the DNS traffic was made by Android and not Monerujo itself.
I will try with and without DoH.

- **Monerujo**
I actually don't know yet what to test on the app, but if I will have the same issues I will test the official CLI or GUI on a PC.

If you think I should test other things too, let me know!
